crates/uv-cli/src/lib.rs

```diff
@@ -76,6 +76,20 @@ pub struct Cli {
     #[command(subcommand)]
     pub command: Box<Commands>,
 
+    #[command(flatten)]
+    pub top_level: TopLevelArgs,
+}
+
+#[derive(Parser)]
+#[command(propagate_version = true)]
+#[command(
+    after_help = "Use `uv help` for more details.",
+    after_long_help = "",
+    disable_help_flag = true,
+    disable_help_subcommand = true,
+    disable_version_flag = true
+)]
+pub struct TopLevelArgs {
```

Comment:
This was admittedly quickly thrown together and the attributes are probably overdone here.